app/views/papers/index.html.erb

```diff
@@ -1,29 +1,65 @@
+<%= stylesheet_link_tag 'papers', media: 'all', 'data-turbolinks-track': 'reload' %>
+
 <p id="notice"><%= notice %></p>
 
-<h1>Papers</h1>
+<p id="papers">Papers</p><br>
+
+<%= form_tag papers_path, { :method => 'get', class: "container", role: "search" }  do %>
+	<div class="row">
+      <div class="col-md-6">
+				<h4>Title</h4>
+          <div id="custom-search-input">
+              <div class="input-group col-md-12">
+                  <div class="form-group">
+                     <%= text_field_tag :search, "", class: "form-control", placeholder: "Title" %>
+                   </div>
+                    <span class="input-group-btn">
+                      <button class="btn btn-info btn-lg" type="button">
+                          <i class="glyphicon glyphicon-search"></i>
+                      </button>
+                    </span>
+                </div>
+            </div>
+        </div>
+	</div>
+<% end %><br><br>
 
-<table>
+<div style="margin-left: 1095px">
+  <%= link_to 'Create Paper', new_paper_path, class: "btn btn-default outline" %>
+  <%= link_to 'Export', new_paper_path, class: "btn btn-default outline" %>
+</div><br>
+
+<table class="table table-bordered table-hover" id="table">
   <thead>
-    <tr>
-      <th>Author</th>
-      <th>Title</th>
-      <th colspan="3"></th>
+    <tr id="header_names">
+      <th scope="col">Authors</th>
+      <th scope="col">Title</th>
+      <th scope="col">Published at</th>
+      <th scope="col">Actions</th>
     </tr>
   </thead>
 
   <tbody>
     <% @papers.each do |paper| %>
       <tr>
-        <td><%= paper.authors.join(",") %></td>
-        <td><%= paper.title %></td>
-        <td><%= link_to 'Show', paper %></td>
-        <td><%= link_to 'Edit', edit_paper_path(paper) %></td>
-        <td><%= link_to 'Destroy', paper, method: :delete, data: { confirm: 'Are you sure?' } %></td>
+        <td scope="row"><% paper.authors.each do |author| %>
+              <%= author.name %><br>
+            <% end %>
+        </td>
+        <td scope="row"><%= paper.title %></td>
+        <td scope="row"><%= paper.created_at %></td>
+        <td width="20%">
+          <div class="btn-group" id="button_tool">
+            <%= link_to 'Show', paper, class: "btn btn-default outline" %>
+            <%= link_to 'Edit', edit_paper_path(paper), class: "btn btn-default outline" %>
+            <%= link_to 'Destroy', paper, method: :delete, data: { confirm: 'Are you sure?' }, class: "btn btn-danger" %>
+          </div>
+        </td>
       </tr>
     <% end %>
   </tbody>
 </table>
 
-<br>
-
-<%= link_to 'New Paper', new_paper_path %>
+<div style="margin-left: 165px">
```

Comment:
Avoid inline CSS: these properties should be in classes or IDs defined in your .scss files